packages/wallets/src/features/cfd/screens/ResubmitPOA/ResubmitPOA.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { useSettings, useStatesList } from '@deriv/api';
+import { InlineMessage, WalletDropdown, WalletText, WalletTextField } from '../../../../components/Base';
+import './ResubmitPOA.scss';
+
+const ResubmitPOA: React.FC = () => {
+    const { data } = useSettings();
+    const country = data?.country_code || '';
+    const { data: statesList } = useStatesList(country);
+
+    const [selectedState, setSelectedState] = useState('');
+
+    const handleSelect = (value: string) => {
+        setSelectedState(value);
+    };
+
+    return (
+        <div className='wallets-resubmit-poa'>
+            <div className='wallets-resubmit-poa__address'>
+                <div className='wallets-resubmit-poa__address__label'>
```

Comment:
Can we not nest the classnames 🐱

Reply:
In this case we have to since its different for mobile and desktop

Reply:
in BEM, generally it is recommended to avoid nesting classnames, 

so probably better .wallest-resubmit-poa__adress-label (almost the same, but flat, and flat strcutres are better as rule of thumb whenever it makes sense)